eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -48,6 +48,8 @@ stages:
               condition: and(succeeded(), ne(variables['Skip.PublishPackage'], 'true'))
               environment: pypi
               dependsOn: TagRepository
+              variables:
+                devFeedName: public/azure-sdk-for-python
```

Comment:
If we promote this to a template parameter we can share it for here and in the dev feed publishing step.